.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.expand": false,
+  "explorer.fileNesting.patterns": {
+    ".prettierrc.json": ".*, *.md, LICENSE"
+  },
+
+  // Hides directories/files from sidebar.
+  // Excludes them from search by default, as well as from quick open.
+  "files.exclude": {
+    // All languages but English
+    "_locales/[abcdfghijklmnopqrstuvwxyz]*": true,
+    "_locales/[abcdefghijklmnopqrstuvwxyz][abcdefghijklmopqrstuvwxyz]*": true,
+    // All languages but English
+    "addons-l10n/[abcdfghijklmnopqrstuvwxyz]*": true,
+    "addons-l10n/[abcdefghijklmnopqrstuvwxyz][abcdefghijklmopqrstuvwxyz]*": true
+  },
+
+  "search.exclude": {
+    "libraries/**": true
```

Comment:
This should only exclude thirdparty libs

Reply:
Okay. I'll exclude the license texts as well.